packages/manager/apps/dedicated/client/app/billing/confirmTerminate/billing-confirmTerminate.html

```diff
@@ -1,218 +1,58 @@
-                                scope="col"
-                                data-translate="terminate_service_services_name"
-                            ></td>
-                            <td
-                                scope="col"
-                                data-translate="terminate_service_services_renew"
-                            ></td>
-                            <td
-                                scope="col"
-                                data-translate="terminate_service_services_frequency"
-                            ></td>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <tr>
-                            <th
-                                scope="row"
-                                data-translate="{{ 'terminate_service_type_' + TerminateServiceCtrl.serviceInfos.serviceType }}"
-                            ></th>
-                            <td
-                                data-ng-bind="TerminateServiceCtrl.serviceInfos.domain"
-                            ></td>
-                            <td>
-                                <billing-renew-date
-                                    data-service-infos="TerminateServiceCtrl.serviceInfos"
-                                ></billing-renew-date>
-                            </td>
-                            <td>
-                                <billing-renew-label
-                                    data-service-infos="TerminateServiceCtrl.serviceInfos"
-                                ></billing-renew-label>
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-
-            <div
-                class="mt-5"
-                data-ng-if="!TerminateServiceCtrl.loading &&
-                             TerminateServiceCtrl.serviceState !== 'suspending' &&
-                             TerminateServiceCtrl.serviceState !== 'suspended'"
+<oui-page-header
+    data-heading="{{:: 'billing_confirm_termination_resiliate' | translate:{
+        serviceName: $ctrl.service.name
+    } }}"
+></oui-page-header>
+
+<div class="container-fluid mt-3">
+    <div class="mt-3" data-ovh-alert></div>
+
+    <oui-message data-type="warning" data-ng-if="$ctrl.hasSpecialConditions()"
+        ><span
+            data-translate="{{:: 'billing_confirm_termination_information_text_' + $ctrl.user.ovhSubsidiary }}"
+        ></span
+    ></oui-message>
+    <oui-message data-type="warning" data-ng-if="!$ctrl.hasSpecialConditions()"
+        ><span
+            data-translate="billing_confirm_termination_information_text"
+        ></span
+    ></oui-message>
+
+    <form name="termination" novalidate class="mt-3">
+        <oui-field
+            data-ng-repeat="question in $ctrl.questions track by question.question"
+            data-label="{{:: 'billing_confirm_termination_question_' + question.question  | translate}}"
```

Comment:
Questions have questions inside them ? 😱 

Is this inception ?

Reply:
Property in payload is `question`